src/eventSchemas/vue/searchSchemas.ts

```diff
@@ -138,8 +139,7 @@ export async function createSearchSchemasWebView(context: ExtContext, node: Regi
         const error = err as Error
         logger.error('Error searching schemas: %O', error)
     } finally {
-        // TODO make this telemetry actually record failures
-        recordSchemasSearch({ result: webviewResult })
+        telemetry.schemas_search.emit({ result: webviewResult })
```

Comment:
the TODO was removed; is the failure recorded in the span now?

Reply:
The comment was stale as apparently I added the correct result a long time ago
To get the failure in the span we need to start using `run` everywhere